sentry/src/main/java/io/sentry/metrics/IMetricsHub.java

```diff
@@ -0,0 +1,24 @@
+package io.sentry.metrics;
+
+import java.util.Map;
+import org.jetbrains.annotations.ApiStatus;
+import org.jetbrains.annotations.NotNull;
+
+@ApiStatus.Internal
+public interface IMetricsHub {
+  /** Captures one or more metrics to be sent to Sentry. */
+  void captureMetrics(final @NotNull EncodedMetrics metrics);
+
+  @NotNull
+  Map<String, String> getDefaultTagsForMetric();
```

Comment:
you can put it in the `MetricsAggregator`, as it just reads values from the options. Also, since we are moving metrics away from the Hub, it would make more sense to decouple them as much as possible